src/pages/home/release-notes/index.md

```diff
@@ -7,49 +7,67 @@ Keywords:
 
 # Release notes
 
+## March 11, 2025
+
+### Adobe Journey Optimizer - Decisioning extension 1.1.0
```

Comment:
I don't think we add tags/launch extension release notes here. Having said that we should still add a note to the user about launch and SDK version required to use this feature.